diff --git a/node_modules/@angular/ssr/fesm2022/ssr.mjs b/node_modules/@angular/ssr/fesm2022/ssr.mjs
index c2e7ff7..6d8fb20 100644
--- a/node_modules/@angular/ssr/fesm2022/ssr.mjs
+++ b/node_modules/@angular/ssr/fesm2022/ssr.mjs
@@ -2585,7 +2585,7 @@ class AngularAppEngine {
      * @param options Options for the Angular server application engine.
      */
     constructor(options) {
-        this.allowedHosts = new Set([...(options?.allowedHosts ?? []), ...this.manifest.allowedHosts]);
+        this.allowedHosts = new Set([...(options?.allowedHosts ?? []), ...(Array.isArray(this.manifest.allowedHosts) ? this.manifest.allowedHosts : [])]);
     }
     /**
      * Handles an incoming HTTP request by serving prerendered content, performing server-side rendering,
