diff --git a/node_modules/@angular/ssr/fesm2022/ssr.mjs b/node_modules/@angular/ssr/fesm2022/ssr.mjs
index bb5c089..d8d26a5 100644
--- a/node_modules/@angular/ssr/fesm2022/ssr.mjs
+++ b/node_modules/@angular/ssr/fesm2022/ssr.mjs
@@ -2599,7 +2599,7 @@ class AngularAppEngine {
         this.trustProxyHeaders = normalizeTrustProxyHeaders(options?.trustProxyHeaders);
     }
     getAllowedHosts(options) {
-        const allowedHosts = new Set([...(options?.allowedHosts ?? []), ...this.manifest.allowedHosts]);
+        const allowedHosts = new Set([...(options?.allowedHosts ?? []), ...(Array.isArray(this.manifest.allowedHosts) ? this.manifest.allowedHosts : [])]);
         if (allowedHosts.has('*')) {
             // eslint-disable-next-line no-console
             console.warn('Allowing all hosts via "*" is a security risk. This configuration should only be used when ' +
